experiments/models.py

```diff
@@ -2,26 +2,57 @@
 
 from hashlib import sha1
 from importlib import import_module
-
 from django.conf import settings
 from django.db import models
-from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import gettext_lazy as _
+
+try:
+    from wagtail.models import Page
+except ImportError:  # fallback for Wagtail <5.0
+    try:
+        from wagtail.core.models import Page
+    except ImportError:  # fallback for Wagtail <2.0
```

Comment:
Drive-by comment: Wagtail < 2 is looooong unsupported. In fact only Wagtail 4.1+ are supported - https://endoflife.date/wagtail. So I suggest you simplify things in that regard. Ditto for Django versions (3.2+)

Reply:
Great point ... it would make the code more readable. 

I thought this pull request might be accepted more readily if I kept the changes to a minimum.  The next updates, I'll favor a steam lined version.